### ISSUE TITLE
Battery Percentage not accurate

### ISSUE THREAD

**Issue by MaxWinterstein** [matijse/eufy-ha-mqtt-bridge#40]
The battery percentage (see #22) seems to be a little off. 

12 days ago it said:  
![image](https://user-images.githubusercontent.com/5927148/106727026-5748e300-660b-11eb-90af-d61698ef999f.png)

today it says:  
![image](https://user-images.githubusercontent.com/5927148/106727079-64fe6880-660b-11eb-8155-aa36849965eb.png)

even ioBroker and the bropat thing stays at the same level:  
![image](https://user-images.githubusercontent.com/5927148/106727467-ce7e7700-660b-11eb-8968-021b724aaab2.png)

Eufy App itself thinks the value should be a little lower:  
![image](https://user-images.githubusercontent.com/5927148/106727716-0a194100-660c-11eb-879f-e35c6804d556.png)

So in conclusion i guess the current power level is not shared towards the eufy server permanently. Maybe just on notification?

Even this seems not to be easily fixed on this code side, we might wan't to drop some note about that.

**Comment by matijse** [comment]
Hmm weird... It should retrieve an updated value every hour. Can you check your log file, to see if the following messages are repeated every hour:

```
Refreshing devices...
Publishing battery percentage ${percentage}% for ${deviceSN}    (1 line for every device)
```

If that is the case, the API is reporting the wrong value. This might be due to the Eufy app getting the value directly from the Camera/HomeBase, while it is sent to the cloud API less often.

If you only see the Publishing battery percentage messages, and not "Refreshing devices...", or no message at all, there may be a bug that information isn't refreshed

**Comment by MaxWinterstein** [comment]
Logs are fine and messages appear every hour:
```
debug: Publishing battery percentage 41% for T8113NXXXXXXXXX {"timestamp":"2021-02-03T21:21:28.966Z"}
debug: Publishing battery percentage 96% for T8113NYYYYYYYYY {"timestamp":"2021-02-03T21:21:28.967Z"}
```

But, as stated here: https://community.home-assistant.io/t/eufy-camera-integration/121758/780 it seems to be updated too rarely.

**Comment by GentleGiantSoftware** [comment]
Noticed that battery values were not updated for the last 9 days at least, although i see the publishing message every hour, battery values have not changed. Did a restart, same old and incorrect values are reported, not the current values shown in the app. Even cleared the docker and updated to the latest version (was one behind), after startup still the same old and incorrect battery values are reported.

**Comment by matijse** [comment]
Some others have seen that Eufy doesn't update the battery status to the cloud that often, so the values are delayed. In the current version those values are retrieved from the cloud API.

With the switch to the new library I think these values can be retrieved directly from the HomeBase / cameras, which makes them update more frequently probably... We'll see...

**Comment by pascalkrielen** [comment]
Any update ?

**Comment by RandomUser99** [comment]
I am also having this issue. The battery charge % is showing 86% in HA but on the Eufy app it is at 70%.

There are no `Publishing battery percentage` entries in the add-on log in the UI.
